spec/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
Please write a test for `#recent_posts` in the User model as all custom methods in the model should be tested.